content/en/code_analysis/faq/_index.md

```diff
@@ -80,6 +80,20 @@ the name of the service is part of the path. If two services are present
 in the path, the service name the closest to the filename is selected.
 
 
+### How are teams associated with code violations and libraries?
+
+Teams are associated by either getting the team associated with the service or using
+the `CODEOWNERS` file and its mapping within Datadog.
+
+When Datadog detects the service for your code violation or library, it automatically
+associates the team attached to the service. For example, if the file `domains/ecommerce/apps/myservice/foo.py`
+is associated with `myservice`, then the team `myservice` will be associated to any violation
+detected in this file.
+
+If no services or teams are found, Datadog uses the `CODEOWNERS` [file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)
+file in your repository. The `CODEOWNERS` file determines which team owns a file in your Git provider. You need
```

Comment:
```suggestion
If no services or teams are found, Datadog uses the `CODEOWNERS` [file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) in your repository. The `CODEOWNERS` file determines which team owns a file in your Git provider. You need
```

Reply:
Can we move the destination link to the bottom of the file? Inline links such as this one aren't compatible with our translation workflows.